.github/scripts/end2end/run-e2e-ctst.sh

```diff
@@ -156,4 +156,4 @@ kubectl run $POD_NAME \
       }
     ]
   }
-}' -- ./run "$COMMAND" $WORLD_PARAMETERS --parallel $PARALLEL_RUNS --retry $RETRIES --retry-tag-filter @Flaky
+}' -- ./run "$COMMAND" $WORLD_PARAMETERS --tags @WIP --parallel $PARALLEL_RUNS --retry $RETRIES --retry-tag-filter @Flaky
```

Comment:
left over from testing ?

Reply:
Needs to be remove but I knew it yeah I was just testing it